igor-web/src/main/groovy/com/netflix/spinnaker/igor/gitlabci/client/model/Project.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2017 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.netflix.spinnaker.igor.gitlabci.client.model;
+
+public class Project {
+    private int id;
+    private String path_with_namespace;
```

Comment:
Same here. annotate the field and give it a proper java var name

Reply:
agree